app/globals.css

```
@@ -40,6 +40,10 @@
   .button {
     @apply button__link__base bg-orange px-7 py-3 text-white outline-blue hover:bg-blue focus:bg-orange focus:outline active:bg-blue disabled:cursor-not-allowed disabled:bg-gray;
   }
+
+  .disbled {
```

Hay un pequeño typo aca

```suggestion
  .disabled {
```